.github/workflows/main.yml

```diff
@@ -13,7 +13,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "pypy-3.8", "pypy-3.9", "pypy-3.10"]
+        python-version:
+          [
+            "3.8",
+            "3.9",
+            "3.10",
+            "3.11",
+            "3.12",
+            "pypy-3.8",
+            "pypy-3.9",
+            "pypy-3.10",
+          ]
+        urllib3-version: ["urllib3"]
```

Comment:
- `urllib3-version` is rather misleading for a name, in pip lingo this is a "requirement specifier". How about `urllib3-dependency` or `urllib3-requirement`?
- Value `urllib3>=2` would make things more obvious